### PR TITLE
Adds recipe for pyls-memestra

### PR DESCRIPTION
**Pull request by marimeireles**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by marimeireles** [comment]
@wolfv 
Can you use your super powers? :supervillain: :wolf:

File: recipes/pyls-memestra/log.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python: can't open file 'build-locally.py': [Errno 2] No such file or directory
```
Comment (wolfv) [inline recipes/pyls-memestra/log.txt:1]: what is this file for?

File: recipes/pyls-memestra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c0e7044561d68119d8dc52d9ba425189f689ad515534c0c5d7a74118b04de49e
+
+build:
+  noarch: python
```
Comment (wolfv) [inline recipes/pyls-memestra/meta.yaml:13]: I think you need to specify the entry points from the setup.py in the recipe here as well.
You could use grayskull to automate this.

Comment (wolfv) [reply recipes/pyls-memestra/meta.yaml:13]: e.g.

```
build:
  number: 0
  skip: true  # [win and vc<14]
  entry_points:
    - mamba = mamba.mamba:main
```

Comment (wolfv) [reply recipes/pyls-memestra/meta.yaml:13]: here are more docs concerning noarch packages: 

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

**Pull request merged**